objects/src/accounts/storage/slot.rs

```diff
@@ -53,6 +60,15 @@ impl StorageSlotType {
             _ => false,
         }
     }
+
+    /// Returns the empty [Word] for a value of this type.
+    pub fn empty_word(&self) -> Word {
+        match self {
+            StorageSlotType::Value { .. } => SimpleSmt::<STORAGE_TREE_DEPTH>::EMPTY_VALUE,
+            StorageSlotType::Map { .. } => EMPTY_STORAGE_MAP,
+            StorageSlotType::Array { .. } => EMPTY_WORD,
```

Comment:
I'm not sure if `EMPTY_WORD` is the best value to represent an empty array of any arity. I assumed the value would be the commitment to the array contents.

Reply:
Maybe all of these should be `SimpleSmt::<STORAGE_TREE_DEPTH>::EMPTY_VALUE`? Now that I think of it, we don't insert `SimpleSmt::<STORAGE_TREE_DEPTH>::EMPTY_VALUE` in the constructor.